src/test/java/com/synopsys/integration/alert/channel/azure/boards/environment/AzureBoardsEnvironmentVariableHandlerTestIT.java

```diff
@@ -0,0 +1,87 @@
+package com.synopsys.integration.alert.channel.azure.boards.environment;
+
+import com.synopsys.integration.alert.api.common.model.AlertConstants;
+import com.synopsys.integration.alert.api.common.model.exception.AlertConfigurationException;
+import com.synopsys.integration.alert.channel.azure.boards.database.accessor.AzureBoardsGlobalConfigAccessor;
+import com.synopsys.integration.alert.channel.azure.boards.model.AzureBoardsGlobalConfigModel;
+import com.synopsys.integration.alert.channel.azure.boards.validator.AzureBoardsGlobalConfigurationValidator;
+import com.synopsys.integration.alert.common.rest.AlertRestConstants;
+import com.synopsys.integration.alert.common.rest.model.Config;
+import com.synopsys.integration.alert.descriptor.api.model.ChannelKeys;
+import com.synopsys.integration.alert.environment.EnvironmentProcessingResult;
+import com.synopsys.integration.alert.environment.EnvironmentVariableUtility;
+import com.synopsys.integration.alert.util.AlertIntegrationTest;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mock.env.MockEnvironment;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@AlertIntegrationTest
+class AzureBoardsEnvironmentVariableHandlerTestIT {
+    private final String TEST_APP_ID = String.valueOf(UUID.randomUUID());
+    private final String TEST_CLIENT_SECRET = "testSecret";
+    private final String TEST_ORGANIZATION_NAME = "test-azure";
+
+    @Autowired
+    private AzureBoardsGlobalConfigAccessor azureBoardsGlobalConfigAccessor;
+    @Autowired
+    private AzureBoardsGlobalConfigurationValidator validator;
+
+    private MockEnvironment mockEnvironment;
+    private EnvironmentVariableUtility environmentVariableUtility;
+
+    @BeforeEach
+    @AfterEach
+    public void cleanup() {
+        azureBoardsGlobalConfigAccessor.getConfigurationByName(AlertRestConstants.DEFAULT_CONFIGURATION_NAME)
+            .map(Config::getId)
+            .map(UUID::fromString)
+            .ifPresent(azureBoardsGlobalConfigAccessor::deleteConfiguration);
+        mockEnvironment = new MockEnvironment();
+        environmentVariableUtility = new EnvironmentVariableUtility(mockEnvironment);
+    }
```

Comment:
We should perhaps split this into an init and cleanup methods rather than calling the same one.

Reply:
agreed this should be two methods because the AfterEach the test creates an instance of MockEnvironment and EnvironmentVariableUtility initialized which isn't needed after each test.